api/app/src/routes/medical/document.ts

```diff
@@ -1,86 +1,52 @@
-import { Request, Response } from "express";
-import Router from "express-promise-router";
-import { OK } from "http-status";
 import { queryDocumentsAcrossHIEs } from "../../command/medical/document/document-query";
 import { getDocuments } from "../../external/fhir/document/get-documents";
 import { getPatientOrFail } from "../../command/medical/patient/get-patient";
-import { asyncHandler, getCxIdOrFail, getFromQuery, getFromQueryOrFail } from "../util";
+import { getCxIdOrFail, getFromQuery, getFromQueryOrFail } from "../util";
 import { toDTO } from "./dtos/documentDTO";
 import { downloadDocument } from "../../command/medical/document/document-download";
+import { DocumentService } from "../../fern/generated/api/resources/document/service/DocumentService";
+import { processAPIKey, checkMAPIAccess } from "../middlewares/auth";
+import { reportMedicalUsage } from "../middlewares/usage";
 
-const router = Router();
+export default new DocumentService(
+  {
+    get: async (req, res) => {
```

Comment:
FYI this is missing the `asyncHandler` logic - we should chat about the best way to incorporate it